src/request/sign-swap/SignSwapApi.js

```diff
@@ -395,29 +431,43 @@ class SignSwapApi extends PolygonRequestParserMixin(BitcoinRequestParserMixin(To
                     throw new Errors.InvalidRequestError('Invalid settlement recipient');
                 }
 
-                /** @type {Omit<KeyguardRequest.SepaSettlementInstruction, 'contractId'>} */
+                /** @type {SepaSettlementInstruction} */
                 const settlement = {
                     type: 'sepa',
                     recipient: {
                         name: /** @type {string} */ (
                             this.parseLabel(
-                                /** @type {{name: unknown}} */ (recipient).name,
+                                /** @type {{name: unknown}} */(recipient).name,
                                 false,
                                 `${parameterName}.recipient.name`,
                             )
                         ),
                         iban: this.parseIban(
-                            /** @type {{iban: unknown}} */ (recipient).iban,
+                            /** @type {{iban: unknown}} */(recipient).iban,
                             `${parameterName}.recipient.iban`,
                         ),
                         bic: this.parseBic(
-                            /** @type {{bic: unknown}} */ (recipient).bic,
+                            /** @type {{bic: unknown}} */(recipient).bic,
                             `${parameterName}.recipient.bic`,
                         ),
                     },
                 };
                 return settlement;
             }
+            case 'sinpemovil': {
+                /** @type {SinpeMovilSettlementInstruction} */
+                const settlement = {
+                    type: 'sinpemovil',
+                    phoneNumber: /** @type {string} */ (
+                        this.parseLabel(
+                            /** @type {{phoneNumber: unknown}} */(obj).phoneNumber,
+                            false,
+                            `${parameterName}.phoneNumber`,
+                        )
+                    ),
```

Comment:
Should we do some more elaborate phone number parsing?
Additionally, are only phone numbers from Costa Rica allowed?